redaxo/src/core/lib/console/cache/clear.php

```diff
@@ -11,13 +11,20 @@ class rex_command_cache_clear extends rex_console_command
     protected function configure()
     {
         $this
-            ->setDescription(rex_i18n::msg('delete_cache'))
-            ->setHelp(rex_i18n::msg('delete_cache_description'));
+            ->setDescription('Clears the redaxo core cache');
     }
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $success = rex_delete_cache();
```

Comment:
Rückgabe-Wert von `rex_delete_cache` scheint eine Message zu sein.
Ich denke, die Funktion ist immer als erfolgreich anzusehen, wenn keine echten Fehler/Exceptions auftreten.

Also würde ich eigentlich eher wie vorher die Message ausgeben, bloß über `$io->success()`. Fehler würde ich von symfony/console behandeln lassen.
Dass die Message deutsch ist würde ich allgemein ja lösen, indem wir in der Console die Sprache fest auf englisch setzen.

Reply:
Wenn wir fest auf englisch gestellt haben, sehe ich es auch so

Reply:
Also insgesamt mit den Messages in der Console würde ich so vorgehen:
- rex_i18n fest auf englisch stellen
- Wenn eine Funktion eine Message schon liefert oder so, die verwenden (wenn es passt)
- Aber niemals nur für die Console neue Messages in den Sprachdateien anlegen, sondern dann direkt Englisch verwenden.
- Auch wenn es etwas bereits in den Sprachdateien gibt, bei der direkten Ausgabe die Variante ohne rex_i18n bevorzugen. Damit meine ich deine Änderung oben bei `setDescription`, die Änderung finde ich so gut, selbst wenn i18n dann auf englisch steht.

Reply:
Passt